internal/cephfs/snapshot.go

```diff
@@ -137,6 +137,11 @@ func getSnapshotInfo(ctx context.Context, volOptions *volumeOptions, cr *util.Cr
 }
 
 func protectSnapshot(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, snapID, volID volumeID) error {
+	// If "snapshot-autoprotect" feature is present, The ProtectSnapshot
+	// call should be treated as a no-op.
+	if checkSubvolumeHasFeature("snapshot-autoprotect", volOptions.Features) {
```

Comment:
as `snapshot-autoprotect` is used in 2 places can you define it as const above and use it?

Reply:
Done.